pages/table.ts

```diff
@@ -0,0 +1,28 @@
+import { Locator, Page, expect } from "@playwright/test";
+
+export class TablePage {
+  readonly page: Page;
+
+  readonly table: Locator;
+  readonly tableRows: Locator;
+  readonly awesomnessLVLunicorn: Locator;
+  readonly awesomnessLVLorangutan: Locator;
+  readonly softnessOfTheFoxFur: Locator;
+  readonly whaleSize: Locator;
+
+
+  constructor(page: Page) {
+    this.page = page;
+
+    this.table = page.getByRole("table")
+    this.tableRows = page.locator('[scope="row"]')
+    this.awesomnessLVLunicorn = page.locator("body > div > table > tbody > tr:nth-child(7) > td:nth-child(3)")

```

Comment:
i takto se to da, pres inspector to najit :) Asi by se nasly lepsi lokatory, nez tohle. Napr. muzes najit pomoci getByText a pak zavolat rodice `getByText('Zviratko').locator('..') a jsi na danem radku.. a pak uz za to zase muzes dat neco.. treba `getBySomething()` a najit si to, co chces.. A vzdycky najdes danou vec, i kdyby se menilo poradi.

Reply:
Funguje mi to takto, mělo by to být nezávislé na změně pořadí.    
this.awesomnessLVLunicorn = page.getByText("Jednorožec").locator("..").locator('td[data-table="lvl-awesomeness"]')

Reply:
Přesně takové řešení bych použil. 🤝